### ISSUE TITLE
WCAG: 2.4 Navigable

### ISSUE THREAD

**Issue by StephDriver** [BirkbeckCTP/janeway#4131]
### Requirement
https://www.w3.org/TR/WCAG22/#navigable
> Provide ways to help users navigate, find content, and determine where they are.

#### :x: Bypass blocks
Bypass blocks are not in use and should be.

#### ✅ page titled
All pages have titles which reflect their type and content (e.g. home pages have journal name, article pages have author, article and journal information).

#### :x: focus order
Focus order is sequential in most cases, but this audit identified a few errors where focus order was disrupted.  Firstly by the carousel which needs an accessibility overhaul, and second with the 'how to cite' popups.

#### :x: link purpose in context
Many links were contextual, but there is work to be done to become fully complaint here, in part because online articles have followed their print tradition where referencing includes full address details for web-links.  The audit also identified that ISSN numbers were being interpreted as telephone numbers by iOS Safari, and automatically being turned into links to phone that number.  

#### :x: multiple ways
Our themes have navigation menus, but our sitemaps and search bars are for articles only.  Currently that means there is only one way to locate static pages (via the navigation menu).

#### :x: headings and labels
Our internal page structures do not have a consistent heading levels. Not all our headings are descriptive.

#### ✅ focus visible
Focus was visible in all tests.

#### ✅ focus not obscured
Focus was not found to be obscured during any tests.

#### bypass blocks
- #3932 
- #4225
- #4242 

#### focus order
- #4235
- #4237 

#### link purpose in context
- #3939
- #4084
- #4083
- #4138
- #4149
- #4221
- #4228
- #4220 

#### multiple ways
- #3919
- #3480
- #4231

#### headings and labels
- #4232 
- #4229
